src/_includes/platforms/configure-releases.md

```diff
@@ -7,7 +7,7 @@ A release is a version of your code that you deploy to an environment. When you
 
 After configuring your SDK, setting up releases is a 2-step process:
 
-1. [Create Release and Associate Commits]({%- link _documentation/workflow/releases.md -%}#create-release)
-2. [Tell Sentry When You Deploy a Release]({%- link _documentation/workflow/releases.md -%}#create-deploy)
+1. [Create Release and Associate Commits]({%- link _documentation/workflow/releases/index.md -%}#create-release)
+2. [Tell Sentry When You Deploy a Release]({%- link _documentation/workflow/releases/index.md -%}#create-deploy)
 
 For more information, see [Releases Are Better With Commits](https://blog.sentry.io/2017/05/01/release-commits.html).
```

Comment:
Is there content in Docs we can point to, rather than a blog from 2017? If not, this link works for now, but let's track a `docs me` request to pull this content into Docs, then link to it.